package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "miden-assembly-examples-playground",
   "version": "0.1.0",
-  "homepage": "https://github.com/0xPolygonMiden/examples",
+  "homepage": "https://0xpolygonmiden.github.io/examples/",
```

Comment:
Maybe we should name this `/playground` instead of `/examples`? What do you think?

Reply:
The GitHub Pages repository URL is always https://{userid}.github.io/{reponame} The GitHub Pages URL is based on the GitHub repo URL, and takes the format https://{userid}.github.io/{reponame} . The moment you use it, GitHub Pages will generate a website based on the contents of your repo.

Let's keep it like is for now.

Reply:
I see - yes, I thought it would be more straightforward. I think we can do this from a subdirectory, so once we change the directory structure in #11 we should be able o deploy to `/playground`